src/state/slices/opportunitiesSlice/resolvers/thorchainsavers/index.ts

```diff
@@ -161,15 +172,16 @@ export const thorchainSaversStakingOpportunitiesMetadataResolver = async ({
   }
 
   if (getConfig().REACT_APP_FEATURE_RUNEPOOL) {
+    const marketData = selectMarketDataByAssetIdUserCurrency(state, thorchainAssetId)
```

Comment:
```suggestion
    const runeMarketData = selectMarketDataByAssetIdUserCurrency(state, thorchainAssetId)
```

Reply:
Fixed in [8b4436a](https://github.com/shapeshift/web/pull/7436/commits/8b4436a0bce282cab384b575cd8da2e999ebd008)